ldk/javascript/src/types/oliveHelps/index.d.ts

```diff
@@ -313,8 +313,9 @@ declare namespace OliveHelps {
     | 'select'
     | 'telephone'
     | 'textInput'
-    | 'sectionTitle'
-    | 'dateTimeInput';
+    | 'dateTimeInput'
+    | 'icon'
+    | 'sectionTitle';
```

Comment:
Might as well alphabetize these for posterity / ease of reference between this file and types.js